lib/membrane_http_adaptive_stream/bandwidth_calculator.ex

```diff
@@ -1,6 +1,8 @@
 defmodule Membrane.HTTPAdaptiveStream.BandwidthCalculator do
   @moduledoc """
   Functions to calculate multimedia track bandwidth according to: https://datatracker.ietf.org/doc/html/rfc8216#section-4.3.4.2
+
+  For a single track it comes down to finding a single segment with the highest bitrate, equal to byte_size to duration ratio.
   """
```

Comment:
- I don't think this module should be public
- The description is no longer valid since we don't calculate the bandwidth according to RFC. It would be great to mention the reason for that as well.

Reply:
We do calculate the bandwidth according to the RFC - see the PR description. Since we are calculating bandwidth for a single track it comes down to finding a segment with a maximum size/duration ratio - as stated in the RFC.